### DISCUSSION TITLE
Tollan Gate

### DISCUSSION THREAD

**Discussion by JuggleMage** [Povstalec/StargateJourney#88] [Q&A]
i saw on the site there used to be a tollan address.... is the dimension still there and if so whats  the dialing addresses please?

**Comment by Povstalec** [comment]
I never added a Tollan Address, I only have plans to add Tollan as a Dimension, but it's not in the mod yet.

**Reply by JuggleMage** [reply]
ah ok.. ty  ^_^ .....good job so far btw  :)
